test/testpartner.ts

```diff
@@ -86,16 +96,52 @@ async function main(): Promise<void> {
       lifi: config.LIFI_INIT,
       polygon: config.POLYGON_INIT,
       piratechain: true,
-      thorchain: config.THORCHAIN_INIT
+      thorchain: config.THORCHAIN_INIT,
+      thorchainda: config.THORCHAIN_INIT
     }
-  })
+  }
 
+  let account: EdgeAccount | undefined
+  let context: EdgeContext
+  if (YOLO_DUMP) {
+    const world = await makeFakeEdgeWorld(fakeUsers, {})
+    context = await world.makeEdgeContext({
+      ...contextOpts,
+      allowNetworkAccess: true
+    })
+    account = await context.loginWithKey('bob', loginKey, {
+      pauseWallets: true
+    })
+  } else {
+    context = await makeEdgeContext(contextOpts)
+    if (YOLO_USERNAME == null) {
+      throw new Error('No username')
+    }
+    if (YOLO_KEY != null) {
+      account = await context.loginWithKey(YOLO_USERNAME, YOLO_KEY, {
+        pauseWallets: true
+      })
+    } else if (YOLO_PIN != null) {
+      account = await context.loginWithPIN(YOLO_USERNAME, YOLO_PIN, {
+        pauseWallets: true
+      })
+    } else if (YOLO_PASSWORD != null) {
+      account = await context.loginWithPassword(YOLO_USERNAME, YOLO_PASSWORD, {
+        otpKey: YOLO_OTPKEY,
+        pauseWallets: true
+      })
+    }
+    if (account == null) {
+      throw new Error('No account')
+    }
+  }
   await context.changeLogSettings({
     defaultLogLevel: 'info',
     sources: {}
   })
 
-  const account = await context.loginWithKey('bob', loginKey)
+  // const lk = await account.getLoginKey()
+  // console.log(`Login key: ${lk}`)
```

Comment:
Cleanup?

Reply:
I left that in there since if you want a real account to login quickly, you'll want to get the loginKey and uncommenting those lines is how you get it

Reply:
Add a comment similar to this then:

```js
// if you want a real account to login quickly, you'll want to get the loginKey and uncommenting those lines is how you get it
```